technic/tools/init.lua

```diff
@@ -1,24 +1,21 @@
 local path = technic.modpath.."/tools"
 
-if technic.config:get_bool("enable_mining_drill") then
-	dofile(path.."/mining_drill.lua")
+local function enabled(name)
+	return technic.config:get_bool("enable_" .. name)
 end
```

Comment:
Opinions about this?
Added to make list of loadable files cleaner and bit easier to read.

Reply:
Makes sense to me.